tools/pubsys/src/repo.rs

```diff
@@ -1,6 +1,9 @@
 //! The repo module owns the 'repo' subcommand and controls the process of building a repository.
 
+pub(crate) mod check_expirations;
+pub(crate) mod refresh_repo;
```

Comment:
We talked about this offline -- I think it's worth seeing how hard it'd be to update `repo.rs` to handle the case of updating a repo without adding any versions/targets, rather than adding a whole new module.  There's a lot of shared process, so it'd be nice not to duplicate it, but if it's hard to combine the two then we can stick with the separate module for now and maybe come back to it later.

(Maybe something like a flattened sub-struct in `RepoArgs` that has the ~5 arguments that get used when adding a version, and if those aren't present, we refresh the repo rather than adding a new version.)

Reply:
I tried to refactor `repo.rs` to accommodate refreshing repo metadata without adding targets or images and I couldn't get it to a state that I'm happy with. The changeset became too big to manage in this PR and would impact original repo creation functionality. So I'd like to create a separate issue to follow up with this if that's OK.

Reply:
Sure, we can also just drop it; the commands are different enough.